File: typical90/Perl/020.pl
#!/usr/bin/perl

my ($a, $b, $c) = split(' ', <STDIN>);
chomp($a, $b, $c);

my $x = 1;
for my _ (0..$b) {
    $x *= $c;
}
print(($a < $x ? "Yes" : "No"));

File: typical90/Swift/020.swift
let input = readLine()!.split(separator: " ").map{Int($0)!}
let (a, b, c) = (
    input[0],
    input[1],
    input[2]
)

let x = 1
for _ in 0 ..< b {
    x *= c
}
print((a < x ? "Yes" : "No"))
